airflow-core/src/airflow/api_fastapi/core_api/datamodels/ui/config.py

```diff
@@ -35,3 +36,4 @@ class ConfigResponse(BaseModel):
     dashboard_alert: list[UIAlert]
     show_external_log_redirect: bool
     external_log_name: str | None = None
+    plugins_extra_menu_items: list[AppBuilderMenuItemResponse]
```

Comment:
can this be defaulted to an empty list?. if yes, we don't have to do `plugins_manager.flask_appbuilder_menu_links or []` in config.py. instead, it could be just `plugins_manager.flask_appbuilder_menu_links`